zou/app/blueprints/source/csv/casting.py

```diff
@@ -67,10 +78,29 @@ def import_row(self, row, project_id):
             target_id = self.asset_map.get(target_key, None)
 
         if asset_id is not None and target_id is not None:
+            entity = entities_service.get_entity_raw(target_id)
             link = breakdown_service.get_entity_link_raw(target_id, asset_id)
             if link is None:
                 breakdown_service.create_casting_link(
                     target_id, asset_id, occurences, label
                 )
+                entity.update({"nb_entities_out": entity.nb_entities_out + 1})
             else:
                 link.update({"nb_occurences": occurences, "label": label})
+            breakdown_service.refresh_shot_casting_stats(entity.serialize())
```

Comment:
This line should be after the test condition verifying the shot type.

Reply:
It's done